os/include/tinyara/security_hal.h

```diff
@@ -256,6 +256,22 @@ typedef int (*hal_init)(_IN_ hal_init_param *params);
  */
 typedef int (*hal_deinit)(void);
 
+#ifdef CONFIG_ARM_TRUSTZONE
+/*
```

Comment:
If you want to remain the description, please follow the doxygen format.

Reply:
Mr Kim, I did not get your comment. Can you explain more?

Reply:
We are following doxygen comment format. You can refer to `os/include/sched.h`
If you want to open the api to user, you can follow like `task_create`.
If you want to use only internal usage, you can follow like `task_init` and `task_activate`.

Reply:
ok